functions/src/transact.ts

```diff
@@ -37,44 +31,4 @@ export const transact = newEndpoint({}, async (req, auth) => {
   })
 })
 
-export async function runTxn(
```

Comment:
I had to remove this function bc the emulator was complaining about the transact function in this file.